src/test/scala/io/altoo/akka/serialization/kryo/BasicSerializationTest.scala

```diff
@@ -16,123 +16,116 @@ limitations under the License.
 
 package io.altoo.akka.serialization.kryo
 
-import akka.actor.{ActorRef, ActorSystem}
+import akka.actor.{Actor, ActorRef, ActorSystem, Props}
 import akka.serialization.{Serialization, _}
 import com.esotericsoftware.minlog.Log
 import com.typesafe.config.ConfigFactory
 import io.altoo.akka.serialization.kryo.serializer.scala.ScalaVersionRegistry
+import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-import scala.concurrent.duration._
+object BasicSerializationTest {
+
+  private val config =
+    s"""
+       |akka {
+       |  loggers = ["akka.event.Logging$$DefaultLogger"]
+       |  loglevel = "WARNING"
+       |}
+       |
+       |akka.actor.serializers {
+       |  kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
+       |}
+       |
+       |altoo.akka.kryo {
+       |  trace = true
+       |  id-strategy = "incremental"
+       |  implicit-registration-logging = true
+       |  post-serialization-transformations = off
+       |  mappings {
+       |    "akka.actor.ActorRef" = 20
+       |    "akka.actor.DeadLetterActorRef" = 21
+       |    "${ScalaVersionRegistry.immutableHashMapImpl}" = 30
+       |    "[L + ${ScalaVersionRegistry.immutableHashMapImpl};" = 31
+       |    "scala.collection.immutable.TreeMap"                = 32
+       |    "[Lscala.collection.immutable.TreeMap;"             = 33
+       |    "${ScalaVersionRegistry.immutableHashSetImpl}" = 34
+       |    "scala.collection.immutable.$$colon$$colon" = 35
+       |    "[J" = 50
+       |    "[D" = 51
+       |    "[Z" = 52
+       |    "[Ljava.lang.Object;" = 53
+       |    "[Ljava.lang.String;" = 54
+       |    "scala.math.Ordering$$String$$" = 100
+       |  }
+       |}
+       |
+       |akka.actor.serialization-bindings {
+       |  "scala.Product" = kryo
+       |  "scala.collection.Map" = kryo
+       |  "scala.collection.Set" = kryo
+       |  "${ScalaVersionRegistry.immutableHashMapImpl}" = kryo
+       |  "${ScalaVersionRegistry.immutableHashSetImpl}" = kryo
+       |  "scala.collection.immutable.TreeMap" = kryo
+       |  "[Ljava.lang.Object;" = kryo
+       |  "akka.actor.ActorRef" = kryo
+       |}
+       |""".stripMargin
+
+  private val configWithCompression =
```

Comment:
could use Config.parser and Config already here (instead of string) and use config with only overridden post-serialization-transformation
->easier to spot diff and shorter...